producers/pipsafety/BUILD

```diff
@@ -0,0 +1,34 @@
+subinclude("@pleasings//docker")
+
+go_binary(
+    name = "python_pipsafety",
+    srcs = [
+        "main.go",
+    ],
+    deps = [
+        "//api/proto:v1",
+        "//producers",
+    ],
+)
+
+go_test(
+    name = "python_pipsafety-test",
+    srcs = [
+        "main.go",
```

Comment:
A nice trick here. You can include `:main` instead listing all the srcs from the `go_binary`.